readme.txt

```diff
@@ -235,7 +235,7 @@ The plugin is produced by [Modern Tribe Inc](http://m.tri.be/18uc).
 
 = [M18.05] TBD =
 
-
+* Fix - Make sure the ticket / unticked events does not count events under `trash` [92236]
```

Comment:
I'd recommend adjusting to something like this:
```
* Fix - Prevent trashed events from affecting ticketed/unticketed counts in the wp-admin [92236]
```

Also, the Central ticket [cites this WP.org thread](https://wordpress.org/support/topic/bug-unticketed-events-count-in-admin-counts-trashed-events/) as the source of the bug, so it looks like there's a user to call out with "props" on this readme entry 👍